tests/unit/plugins/modules/cm_autotls/test_cm_autotls.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2025 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+import os
+import logging
+import pytest
+
+from ansible_collections.cloudera.cluster.plugins.modules import cm_autotls
+from ansible_collections.cloudera.cluster.tests.unit import (
+    AnsibleExitJson,
+    AnsibleFailJson,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def conn():
```

Comment:
Update to use the current pytest fixtures: https://github.com/cloudera-labs/cloudera.cluster/blob/devel/tests/unit/plugins/modules/cm_service/test_cm_service.py#L41

Reply:
Done